### PR TITLE
Dark Theme

### LINKED ISSUE #56: Night Theme
Theme with dark palette

### PR DESCRIPTION
**Pull request by wiseodd**
Closes #56 

@timotiusnc

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -5,6 +5,17 @@
...
         <item name="colorPrimaryDark">@color/pumpkin_primaryDark</item>
         <item name="colorAccent">@color/pumpkin_accent</item>
+        <item name="title_color">@color/grey_1000b</item>
+        <item name="dim_title_color">@color/grey_500</item>
+        <item name="list_bg_color">@color/grey_1000w</item>
+        <item name="card_style">@style/CardView</item>
+    </style>
+
+    <style name="DarkPumpkinTheme" parent="Theme.AppCompat.NoActionBar">
+        <item name="title_color">@color/grey_1000w</item>
```
Comment (timotiusnc) [inline app/src/main/res/values/styles.xml:15]: Why doesn't DarkPumpkinTheme have item named `colorPrimary` and `colorAccent` like PumpkinTheme?

File: app/src/main/java/io/pumpkinz/pumpkinreader/data/NewsAdapter.java [hunk 2]
```diff
@@ -80,15 +81,19 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
                 News news = this.dataset.get(position);
                 NewsViewHolder newsViewHolder = (NewsViewHolder) viewHolder;
 
-                int color;
-
                 if (PreferencesUtil.isNewsRead(fragment.getActivity(), news)) {
-                    color = fragment.getResources().getColor(R.color.grey_200);
+                    TypedValue dimTitleColor = new TypedValue();
```
Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/data/NewsAdapter.java]: I think we can save a few lines by extracting the

```
newsViewHolder.getTitle().setTextColor(titleColor.data);
newsViewHolder.getScore().setTextColor(titleColor.data);
```

The `if` part only determines whether `titleColor` is from `R.attr.dim_title_color` or `R.attr.title_color`.

File: app/src/main/java/io/pumpkinz/pumpkinreader/util/ColorUtil.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package io.pumpkinz.pumpkinreader.util;
```
Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/util/ColorUtil.java]: Is this util used somewhere?

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (wiseodd) [inline app/src/main/res/values/styles.xml:15]: Inheritance

**Comment by wiseodd** [comment]
Updated

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (timotiusnc) [inline app/src/main/res/values/styles.xml:15]: What about this one :(?

**Pull request merged**